codersdk/workspaces.go

```diff
@@ -66,6 +66,7 @@ type Workspace struct {
 	AllowRenames     bool             `json:"allow_renames"`
 	Favorite         bool             `json:"favorite"`
 	NextStartAt      *time.Time       `json:"next_start_at" format:"date-time"`
+	IsPrebuild       bool             `json:"is_prebuild"`
```

Comment:
suggestion: Add a comment to the field so that it's clear what this means to consumers.

Reply:
ℹ️  Just a heads up: I'm not sure what is the naming convention in the BE, if we have any, for boolean values. I see you are using the "Is" prefix, but the "Favorite" attribute does not.

@johnstcn wdyt?